tests/integration.rs

```diff
@@ -736,3 +736,53 @@ fn test_expire() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_alloc_data_type() -> Result<()> {
+    let port: u16 = 6503;
+    let _guards = vec![start_valkey_server_with_module("data_type", port)
```

Comment:
Is the name of the Module `alloc` or `data_type`?

Reply:
It's [alloc](https://github.com/valkey-io/valkeymodule-rs/blob/main/examples/data_type.rs#L85C5-L85C9) - I'll update it to be clear, thanks for the feedback.

Reply:
Thank you for the clarification. That makes sense